packages/graphile-build-pg/src/plugins/introspectionQuery.js

```diff
@@ -9,9 +9,19 @@
  */
 function makeIntrospectionQuery(
   serverVersionNum: number,
-  options: { pgLegacyFunctionsOnly?: boolean } = {}
+  options: {
+    pgLegacyFunctionsOnly?: boolean,
+    pgIgnoreRBAC?: boolean,
+  } = {}
 ): string {
-  const { pgLegacyFunctionsOnly } = options;
+  const { pgLegacyFunctionsOnly, pgIgnoreRBAC } = options;
```

Comment:
```suggestion
  const { pgLegacyFunctionsOnly = false, pgIgnoreRBAC = true } = options;
```

Reply:
NP.. I'm playing in your sandbox now, gotta follow your rules ;)